ansible/roles/system/files/010_pi-nopasswd

```diff
@@ -0,0 +1 @@
+pi ALL=(ALL) NOPASSWD: ALL
```

Comment:
This is wrong. We should require password. We then need to whitelist selected few commands.

Reply:
Fair enough, but we still need another one then for the locked down state.

Reply:
The task above just deletes this file (010_pi-nopasswd file for disabling sudo password in raspbian by default)